src/graphql/resolvers/ssb.js

```diff
@@ -42,6 +123,110 @@ module.exports = function Resolvers () {
     })
   }
 
+  /**
+   * Gets all the votes on a message
+   * TODO: doesnt handle unlikes and duplicates yet.
+   * @param {string} id - id of a message
+   */
+  const getVotes = (id) => {
+    return new Promise((resolve, reject) => {
+      pull(
+        ssb.db.query(
+          where(
+            votesFor(id)
+          ),
+          descending(), // latest => oldest
+          toPullStream()
+        ),
+        pull.map(msg => {
+          const vote = msg?.value?.content?.vote
+          return {
+            author: msg.value.author,
+            timestamp: msg.value.timestamp,
+            value: vote?.value,
+            expression: vote?.expression
+          }
+        }),
+        pull.collect((err, votes) => {
+          if (err) reject(err)
+          else resolve(votes)
+        })
+      )
+    })
+  }
+
+  /**
+   * Gets all the threads initiated by a certain profile id 
+   * @param {string} id - id of a profile
+   * @param {object} opts - optional parameters
+   * @param {int} [opts.limit=10] - max amount of threads to return
+   * @param {int} opts.threadMaxSize - max amount of messages in each thread to return
+   */
+  const getThreads = (id, opts) => {
```

Comment:
I think we should call this `feedId` to make it more specific.
And `msgId` on the votes one